CONTRIBUTING.md

```diff
@@ -11,19 +11,19 @@ If you have questions about how to install or use ownCloud, please direct these
   - Go to one of the repositories, click "issues" and type any word in the top search/command bar.
   - You can also filter by appending e. g. "state:open" to the search string.
   - More info on [search syntax within github](https://help.github.com/articles/searching-issues)
-* This repository ([notes](https://github.com/owncloud/notes/issues)) is *only* for issues within the ownCloud notes code.
-* __SECURITY__: Report any potential security bug to security@owncloud.com following our [security policy](https://owncloud.org/security/) instead of filing an issue in our bug tracker
+* This repository ([notes](https://github.com/nextcloud/notes/issues)) is *only* for issues within the nextcloud notes code.
+* __SECURITY__: Report any potential security bug to security@nextcloud.com following our [security policy](https://nextcloud.com/security/) instead of filing an issue in our bug tracker
```

Comment:
I was also unsure if that email existed, so decided to change it anyway. If anyone knows better please just tell me.